jmix-charts/charts-flowui-kit/src/main/resources/META-INF/frontend/src/chart/jmix-chart.js

```diff
@@ -63,6 +63,16 @@ class JmixChart extends ResizeMixin(ElementMixin(PolymerElement)) {
         }
     }
 
+    static get forwardedEventNames() {
+        return ['click', 'dblclick', 'mousedown', 'mousemove', 'mouseup', 'mouseover', 'mouseout', 'globalout',
+            'highlight', 'downplay', 'selectchanged', 'legendselectchanged', 'legendselected', 'legendunselected',
+            'legendselectall', 'legendinverseselect', 'legendscroll', 'datazoom', 'datarangeselected', 'graphroam',
+            'georoam', 'treeroam', 'timelinechanged', 'timelineplaychanged', 'restore', 'dataviewchanged',
+            'magictypechanged', 'geoselectchanged', 'geoselected', 'geounselected', 'axisareaselected',
+            'brush', 'brushend', 'brushselected', 'globalcursortaken', 'rendered', 'finished',
+        ];
+    }
```

Comment:
Unused events have not been removed and are still sent by the client-size.

- Timeline
- TreeRoam
- GraphRoam
- Geo…
- DataViewChanged

Reply:
removed